charts/images.yaml

```diff
@@ -259,7 +260,7 @@ images:
 # Shoot optional addons
 - name: kubernetes-dashboard
   sourceRepository: github.com/kubernetes/dashboard
-  repository: kubernetesui/dashboard
+  repository: mirror.gcr.io/kubernetesui/dashboard
```

Comment:
As discussed in the other PR, we probably should not rely on the GCR mirror and rather push these images to the `gardener` registry.

Reply:
tracked in https://github.com/gardener/ci-infra/issues/619

Reply:
The image switch can be done in a dedicated PR to unblock this one.
Continuing with docker hub images is fine – at least for now. The local kind seed basically has dual-stack internet access and is able to pull images from docker hub.
Later on, when working on the IPv6 shoot workers this won't work, but we will finish https://github.com/gardener/ci-infra/issues/619 by then probably.

Hence, I will drop the related changes from this PR.